src/VBTestCode/VBTestCode.vbproj

```diff
@@ -4,7 +4,7 @@
     <TargetFrameworks>net452;netcoreapp2.0</TargetFrameworks>
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>$(SolutionDir)Test.snk</AssemblyOriginatorKeyFile>
-    <Optimize>false</Optimize>
+    <Optimize>true</Optimize>
```

Comment:
You should just drop this instead.

Reply:
Is true the default value?

Reply:
The default is configuration-specific. For Debug it's false, and for Release it's true. So, you should drop it! 😄